mllib/src/main/scala/org/apache/spark/ml/feature/RFormula.scala

```diff
@@ -144,6 +145,12 @@ class RFormulaModel private[feature](
     pipelineModel: PipelineModel)
   extends Model[RFormulaModel] with RFormulaBase {
 
+  /** @group setParam */
```

Comment:
Right, we could change `featuresCol` in linear models. But the underlying model is a pipeline. It might be hard to change.